taffybar.cabal

```diff
@@ -83,6 +84,7 @@ library
                , transformers >= 0.3.0.0
                , transformers-base >= 0.4
                , tuple >= 0.3.0.2
+               , udev == 0.1.0.0
```

Comment:
@LeifW you need to add udev to the stack extra deps. Make sure to get the stack.yaml.example file too

Reply:
This will restrict taffybar to Linux-only, as this is a binding to libudev. Did we want to make e.g. a separate taffybar-extras or taffybar-linux for this sort of thing, or hide that extra dep behind a configure flag and only add it conditionally (like xmonad-extras does for its extra deps?)

Reply:
I'm fine with either